helm-chart/amalthea/crd-spec.yaml

```diff
@@ -221,5 +221,14 @@ spec:
             Size of the PVC or sizeLimit of the emptyDir volume which backs the
             session respectively.
           type: string
+        emptyDir:
+          type: object
+          description: "Options related to using emptyDir for storage. If PVC storage is disabled then emptyDir storage is used, otherwise all options here are ignored."
+          default: {}
+          properties:
+            enforceSizeLimit: 
```

Comment:
I'm wondering if we could just interpret 
```
pvc: 
  enabled: false
  size: {null | ""}
```
as setting for an `emptyDir` volume without any size limit. It's a bit shorter and - probably more importantly - we can do it without changing the CRD.